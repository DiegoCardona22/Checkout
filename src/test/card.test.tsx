// @packages
import { shallow } from "enzyme";

// @scripts
import Card from "src/components/atoms/card";

describe("Testing the component <Card />", () => {
  const id = "card";
  const image = "https://via.placeholder.com/1920x1080";
  const description = "Card Description";
  const title = "Card Title";
  let wrapper = shallow(
    <Card
      description={description}
      id={id}
      imageUrl={image}
      title={title}
    />
  );

  beforeEach(() => {
    wrapper = shallow(
      <Card
        description={description}
        id={id}
        imageUrl={image}
        title={title}
      />
    );
  });

  it("Should render the component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
