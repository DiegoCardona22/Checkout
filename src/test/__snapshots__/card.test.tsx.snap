// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the component <Card /> Should render the component correctly 1`] = `ShallowWrapper {}`;
